frontend/components/Player/Player.jsx

```diff
@@ -0,0 +1,13 @@
+const Player = ({ player }) => {
+    return (
+        <div>
+            <h1>Members</h1>
+            {player.map((member, idx) => {
+                let email = member.email;
+                return <div key={`${email}${idx}`}>{email}</div>;
```

Comment:
You'll often hear that using the index in key is bad practice. In this case, you'll never have 2 emails that are the same, so you should be safe to just use the email as the key.